subsys/bluetooth/host/classic/a2dp.c

```diff
@@ -282,8 +282,10 @@ static void bt_a2dp_media_data_callback(struct bt_avdtp_sep *sep, struct net_buf
 	stream = ep->stream;
 	media_hdr = net_buf_pull_mem(buf, sizeof(*media_hdr));
 
+	TOOLCHAIN_DISABLE_WARNING(TOOLCHAIN_WARNING_ADDRESS_OF_PACKED_MEMBER);
 	stream->ops->recv(stream, buf, sys_get_be16((uint8_t *)&media_hdr->sequence_number),
```

Comment:
those (if I get the code right), could be converted to sys_be16_to_cpu(media_hdr->sequence_number),  that should avoid unaligned access warning